.ci/validation.sh

```diff
@@ -895,6 +895,7 @@ no-exception-struts)
   sed -i'' 's/#apache-struts/apache-struts/' projects-to-test-on.properties
   groovy ./diff.groovy --listOfProjects projects-to-test-on.properties \
       --patchConfig checks-nonjavadoc-error.xml  -p "$BRANCH" -r ../../..  \
+      --useShallowClone \
```

Comment:
There are 13 usages of diff.groovy please update all of them

Reply:
Done